src/adapter.js

```diff
@@ -247,8 +247,10 @@ function KarmaReporter (tc, jasmineEnv) {
         result.log.push(formatFailedStep(steps[i]))
       }
 
-      // Report the name of fhe failing spec so the reporter can emit a debug url.
-      result.debug_url = debugUrl(specResult.fullName)
+      if (window && window.location && window.location.origin) {
```

Comment:
Are we assuming that this code will run in Node?
According to #262 that doesn't seem like the case.
`window` is still defined, but window.location is not, because the error message was
```
Cannot read property 'origin' of undefined
```
when trying to access `window.location.origin`

I'm asking because the check for 'window' done this way will actually throw a ReferenceError in Node

Reply:
Of course I don't know how this error can happen. But you are completely correct, so I used `typeof`.